Sources/Foundation/Documentation.docc/Curation/FoundationEssentials.md

```diff
@@ -0,0 +1,37 @@
+# Foundation Essentials
+
+Access core types and essential functionality used by most applications.
+
+## Overview
+
+The Foundation Essentials package provides the most important types for everyday development:
+
+* Core types beyond those provided by the Swift standard library, such as ``Data``, ``Decimal``, and ``UUID``.
+* Dates and time intervals.
+* An ``AttributedString`` type for strings with runs of metadata, such as styling information.
+* JSON encoding and decoding.
+* A ``URL`` type for network and local resources.
+
+Foundation Essentials provides these features without system dependencies, so developers can count on consistent behavior, regardless of platform.
```

Comment:
I'm not sure if this "without system dependencies" is entirely correct. We still use other libraries. It's just that FoundationEssentials abstracts away the underlying dependencies and chooses the one depending on the platform for developers, so they don't need to worry about platform check.

Reply:
I was trying to include this concept from the [original announcement](https://www.swift.org/blog/future-of-foundation/):

> a new FoundationEssentials package will provide the most important types in Foundation with no system dependencies to help accomplish this.

But yeah, without the context of the rest of that announcement, there are lots of ways to read "without system dependencies".